packages/dd-trace/src/appsec/sdk/set_user.js

```diff
@@ -0,0 +1,30 @@
+'use strict'
+
+const { getRootSpan } = require('./utils')
+const log = require('../../log')
+
+function setUserTags (user, rootSpan) {
+  for (const k of Object.keys(user)) {
+    rootSpan.setTag(`usr.${k}`, '' + user[k])
+  }
+}
+
+function setUser (tracer, user) {
+  if (!user || !user.id) {
+    log.warn('User not available in setUser')
```

Comment:
the existing log in the event SDK is 
```suggestion
    log.warn('Invalid user provided to setUser')
```